plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeGcsConnectorSmokeTest.java

```diff
@@ -124,14 +125,15 @@ protected HiveMinioDataLake createHiveMinioDataLake()
         hadoopCoreSiteXmlTempFile.toFile().deleteOnExit();
         Files.writeString(hadoopCoreSiteXmlTempFile, gcpSpecificCoreSiteXmlContent);
 
-        HiveMinioDataLake dataLake = new HiveMinioDataLake(
-                bucketName,
-                ImmutableMap.of(
+        HiveHadoop.Builder hiveHadoopBuilder = HiveHadoop.builder()
```

Comment:
nit: We can initialize HiveHadoop without a builder variable. Same for Adls test class:
```java
        HiveHadoop hiveHadoop = HiveHadoop.builder()
                .withImage(HIVE3_IMAGE)
                .withNetwork(closeAfterClass(newNetwork()))
                .withFilesToMount(ImmutableMap.of(
                        "/etc/hadoop/conf/core-site.xml", hadoopCoreSiteXmlTempFile.normalize().toAbsolutePath().toString(),
                        "/etc/hadoop/conf/gcp-credentials.json", gcpCredentialsFile.toAbsolutePath().toString()))
                .build();
```